source/dagger/gameplay/ping_pong/ping_pong_main.cpp

```diff
@@ -54,6 +56,7 @@ void PingPongGame::CoreSystemsSetup(Engine& engine_)
     engine_.AddSystem<SpriteRenderSystem>();
     engine_.AddSystem<AnimationSystem>();
     engine_.AddSystem<TransformSystem>();
+    engine_.AddSystem<BallGlowingSystem>();
```

Comment:
move to GameplaySystemsSetup

Reply:
Yea... I thought that I did put it there. It's not working anyway...